app/reducers/traces.js

```diff
@@ -151,6 +151,7 @@ export default function (state = initialState, action) {
       let editedTraces = _cloneDeep(state.editedTraces)
       traces = traces.filter(trace => trace.id !== action.trace.id)
       if (editedTrace.apiId && editedTrace.geojson.properties.description !== action.description) {
+        editedTrace.status = 'pending edit'
```

Comment:
naming question - not sure if this should be something like `pending edit` or `pendingEdit`. Also, it would be nice maybe to put something like `EDIT_TRACE_STATUSES` in a constants?

Reply:
Yeah putting in a constant is probably a good idea. I can do that.

Reply:
Realized that we haven't been doing that and we should but I'm going to merge this and open a ticket to fix that across the project in different PR.